.gitignore

```diff
@@ -4,3 +4,4 @@ dist/
 docs/_build
 
 .coverage
+*.pyc
```

Comment:
Please remove this from the `.gitignore`

Reply:
I'm fine with doing that, but this means there will be a ton of *.pyc files that have to be ignored. For example:

```
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        pydocstringformatter/__pycache__/
        pydocstringformatter/_configuration/__pycache__/
        pydocstringformatter/_formatting/__pycache__/
        pydocstringformatter/_utils/__pycache__/

no changes added to commit (use "git add" and/or "git commit -a")
```

We might also consider ignoring ``__pycache__`` as in:

https://github.com/pandas-dev/pandas/blob/fb754d71db2897b40dd603e880a15e028f64f750/.gitignore#L72

Reply:
Are you opposed to ignoring `__pycache__` in `~/.gitignore`? In general I don't ever see the need to commit those files to `git`.